alf/algorithms/predictive_representation_learner.py

```diff
@@ -72,18 +74,25 @@ def __init__(self,
                 to construct an instance of ``Network`` for decoding. The network
                 should take the latent representation as input and output the
                 predicted value of the target.
-            loss (Callable): loss function with signature ``loss(y_pred, y_true)``.
-                Note that it should not reduce to a scalar. It should at least
-                keep the batch dimension in the returned loss.
+            loss_ctor (Callable): loss function with signature
+                ``loss(y_pred, y_true)``. Note that it should not reduce to a
+                scalar. It should at least keep the batch dimension in the
+                returned loss.
             loss_weight (float): weight for the loss.
             optimizer (Optimzer|None): if provided, it will be used to optimize
                 the parameter of decoder_net
             normalize_target (bool): whether to normalize target.
                 Note that the effect of this is to change the loss. The predicted
                 value itself is not normalized.
+            append_target_field_to_name (bool): whether append target field to
```

Comment:
In this case, why cant the user pass "name.target_field" directly as the argument? Or perhaps default this argument to True to make it a little bit easier to config?

Reply:
Changed default value to True. 
The reason for adding this argument is that ``target_field`` is not yet available when we are constructing the decoders in predictive representation learner.